tests/samples/test_api.py

```diff
@@ -63,13 +66,24 @@ def __init__(self):
         "total_count": 3
     })
 ])
-async def test_find(find, per_page, page, label_filter, d_range, meta, snapshot, spawn_client, static_time):
+async def test_find(find, per_page, page, label_filter, d_range, meta, snapshot, spawn_client, static_time,
+                    test_session):
     client = await spawn_client(authorize=True)
 
     time_1 = arrow.get(static_time.datetime).datetime
     time_2 = arrow.get(static_time.datetime).shift(hours=1).datetime
     time_3 = arrow.get(static_time.datetime).shift(hours=2).datetime
 
+    label_1 = Label(id=1, name="Bug", color="#a83432", description="This is a bug")
+    label_2 = Label(id=2, name="Info", color="#03fc20", description="This is a info")
+    label_3 = Label(id=3, name="Question", color="#0d321d", description="This is a question")
+
+    async with test_session as session:
+        session.add(label_1)
+        session.add(label_2)
+        session.add(label_3)
```

Comment:
Use this?:

https://docs.sqlalchemy.org/en/13/orm/session_api.html#sqlalchemy.orm.session.Session.add_all